src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMessage.cs

```diff
@@ -60,6 +74,14 @@ internal CoseMessage(CoseHeaderMap protectedHeader, CoseHeaderMap unprotectedHea
         [MemberNotNullWhen(false, nameof(Content))]
         internal bool IsDetached => _content == null;
 
+        /// <summary>
+        /// Decodes a CBOR payload as a COSE_Sign1 message.
+        /// </summary>
+        /// <param name="cborPayload">The sequence of CBOR-encoded bytes to decode.</param>
+        /// <returns>A <see cref="CoseSign1Message"/> representation of <paramref name="cborPayload"/>.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="cborPayload"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="cborPayload"/> contains trailing data after COSE_Sign1 was complete.</exception>
+        /// <exception cref="CryptographicException">Decodification failed, see the inner exception for more details.</exception>
```

Comment:
```suggestion
        /// <exception cref="CryptographicException">The specified data did not decode as a COSE_Sign1 value.</exception>
```

Reply:
This is more consistent with the `GetValueAs*` methods that also perform decoding.
```suggestion
        /// <exception cref="CryptographicException"><paramref name="cborPayload"/> could not be decoded as a COSE_Sign1 message.</exception>
```